test/test_torch.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
We don't need this, right?

Reply:
Theoretically not. Just trying to make CI pass.